docs/docs/concepts/dialogue-understanding.mdx

```diff
@@ -21,9 +25,62 @@ In the CALM approach, _Dialogue Understanding_ aims to represent how the end use
 wants to progress the conversation.
 There is currently only one command generator available: the `LLMCommandGenerator`.
 
-## Using the LLMCommandGenerator in Your Bot
+Like its name indicates, the `CommandGenerator` generates "commands" that are then internally processed to
+trigger operations on the current conversation. Here is a reference of all supported commands:
+
+### StartFlowCommand
+
+This command indicates the assistant should start a new [flow](./flows.mdx).
+
+### CancelFlowCommand
+
+This command indicates the assistant should cancel the current [flow](./flows.mdx).
+It powers [Conversation Repair's Cancellation use case](./conversation-repair.mdx#3-cancellations).
+
+### SetSlotCommand
+
+This command indicates the assistant should set a [slot](./domain.mdx#slots) to a given value.
+
+### CorrectSlotsCommand
+
+This command indicates the assistant should change the value of a given [slot](./domain.mdx#slots)
+to a new value. It powers [Conversation Repair's Correction use case](./conversation-repair.mdx#2-corrections).
+
+### ClarifyCommand
+
+This command indicates the assistant should ask for clarification. It powers
+[Conversation Repair's Clarification use case](./conversation-repair.mdx#6-clarification).
+
+### ChitChatAnswerCommand
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+This command indicates the assistant should reply a chitchat style free-form answer.
+It powers [Conversation Repair's Chitchat use case](./conversation-repair.mdx#4-chitchat).
+
+### KnowledgeAnswerCommand
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+This command indicates the assistant should reply a knowledge-based free-form answer. It works
+hand in hand with the [DocSearch policy](./policies.mdx#docsearch-policy).
+
+### HumanHandoffCommand
```

Comment:
@dakshvar22 same question - Pro?

Reply:
The exact behaviour of what happens after this command is actually not thought out at all. But the default behaviour (yet to be implemented / designed) that comes out of the box would be something like trigger `action_default_fallback` which customers can customize. I feel the real value add here would be if we built integrations to customer service platforms like servicenow and having those work out of the box. That would be a good pro feature.

What do you think?

Reply:
yes this should be pro

Reply:
yeah also we could trigger e.g. a summarization to happen first. I think there's more to be done to flesh this out for sure

Reply:
I'll keep this like this until the functionality is designed and implemented and put into pro -- @dakshvar22 I'll let you revisit this then ;)

Reply:
Ticket for designing the functionality [here](https://rasahq.atlassian.net/browse/ENG-579). Will discuss with @andreaklmn and take this further.